From 87028310c3cd5cc39b91ba9743432d1cb49e36d4 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Wed, 12 Apr 2023 09:29:07 +0800
Subject: [PATCH 3/4] dts: Aptiv: modify pfe bmu2 pool address and size for pfe
 slave mode

This patch is based on the patch:
wryocto/layers/meta-aptiv/recipes-kernel/linux/files/0001-dts-s32g-pfe-modify-BMU2-reserved-memory-node-to.patch

Upstream-Status: Pending

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 .../boot/dts/freescale/s32g274a-aptiv-pfems.dts    | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/s32g274a-aptiv-pfems.dts b/arch/arm64/boot/dts/freescale/s32g274a-aptiv-pfems.dts
index 1b05a2761157b..6c863a9a4d997 100644
--- a/arch/arm64/boot/dts/freescale/s32g274a-aptiv-pfems.dts
+++ b/arch/arm64/boot/dts/freescale/s32g274a-aptiv-pfems.dts
@@ -14,6 +14,20 @@
 #include "s32g274a-aptiv.dtsi"
 #include "s32g-pfe-slave.dtsi"
 
+
+/delete-node/ &pfe_reserved_bmu2;
+/ {
+	reserved-memory {
+		pfe_reserved_bmu2: pfebufs@83000000 {
+			compatible = "nxp,s32g-pfe-bmu2-pool";
+			/* BMU2: 2 MB */
+			reg = <0 0x83000000 0 0x200000>;
+			no-map;
+			status = "okay";
+		};
+	};
+};
+
 &pfe_aux0 {
 	status = "okay";
 };
-- 
2.36.1

